src/dropdown/list/listdropdownview.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import DropdownView from '/ckeditor5/ui/dropdown/dropdownview.js';
+
+/**
+ * The basic list dropdown view class.
+ *
+ * @memberOf ui.dropdown
+ * @extends ui.View
+ */
+
+export default class ListDropdownView extends DropdownView {
+	constructor( model ) {
+		super( model );
+
+		this.listenTo( this.model, 'change:isOn', ( evt, name, value ) => {
```

Comment:
The whole cost of these callbacks combined for the whole world in next 10 years will be less than me writing this comment ;).

But ok. Since you already implemented this in this way, let's keep it.